toolkit/scripts/imggen.mk

```
@@ -79,7 +79,7 @@ clean-imagegen:
 # the same, but the actual .rpm files may be fundamentally different.
 $(STATUS_FLAGS_DIR)/imagegen_cleanup.flag: $(depend_REPO_SNAPSHOT_TIME)
 	@echo "REPO_SNAPSHOT_TIME has changed, sanitizing rpm cache"
-	rm -rf $(local_and_external_rpm_cache)
+	find $(local_and_external_rpm_cache) -type f -name '*.rpm' -exec rm -f {} +
```

FYI `find` already has a `-delete` action, so you should be able to shorten this a bit. Also, just to be safe, it's good to surround paths with quotes in case there are any spaces in the path:
```suggestion
	find "$(local_and_external_rpm_cache)" -type f -name '*.rpm' -delete
```